content/docs/command-reference/exp/run.md

```diff
@@ -100,6 +95,13 @@ committing them to the Git repo. Unnecessary ones can be [cleared] with
   > stages may sometimes be executed several times depending on the state of the
   > [run-cache] at that time.
 
+  > ⚠️ `dvc exp run --run-all [--jobs]` is now a shortcut for
+  > `dvc queue start [--jobs]` followed by `dvc queue logs -f`. The `--run-all`
+  > and `--jobs` options will likely be deprecated and removed in a future DVC
+  > release. It is recommended to migrate your workflows to use
+  > `dvc queue start` and `dvc queue logs`. Refer to the `dvc queue`
+  > documentation for more details.
+
```

Comment:
Should we also include this kind of warning in core DVC (i.e. actually logging `DeprecationWarning`)?

Reply:
It's fine to say that this will be deprecated (we are already doing it here). There shouldn't be a hard requirement to include deprecation of experiment features since they are "experimental" in 2.X, but it would be nice.